chromiumcontent/chromiumcontent.gyp

```diff
@@ -33,6 +33,7 @@
         '<(DEPTH)/content/content.gyp:content_app_both',
         '<(DEPTH)/content/content_shell_and_tests.gyp:content_shell_pak',
         '<(DEPTH)/net/net.gyp:net_with_v8',
+        '<(DEPTH)/pdf/pdf.gyp:pdf',
```

Comment:
The component build has been finished now. For pdf.lib static library in component build, we need to link all its dependent libraries manually. Also introducing change on atom/brightray#102.

@zcbenz please take a look.